purchase_requisition_analytic/models/purchase_order.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2023 Moduon Team S.L. <info@moduon.team>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order.line"
+
+    @api.model
+    def create(self, vals):
+        res = super().create(vals)
+        if res.order_id.requisition_id:
+            res.account_analytic_id = res.order_id.requisition_id.analytic_account_id.id
```

Comment:
I wonder if we should let users define their own data instead of overwriting always. Something like this:

```suggestion
        res.account_analytic_id = res.account_analytic_id or res.order_id.requisition_id.analytic_account_id
```

Does that make more sense than the current implementation?